fl4health/parameter_exchange/partial_parameter_exchanger.py

```diff
@@ -0,0 +1,35 @@
+from abc import abstractmethod
+from typing import Optional, Tuple, TypeVar
+
+import torch.nn as nn
+from flwr.common.typing import Config, NDArrays
+
+from fl4health.parameter_exchange.packing_exchanger import ParameterExchangerWithPacking
+
+T = TypeVar("T")
+
+
+class PartialParameterExchanger(ParameterExchangerWithPacking[T]):
```

Comment:
This is a bit of an odd inheritance because `ParameterExhangerWithPacking` inherits from `FullParameterExchanger`, which defines push and pull parameters. That makes setting them `abstractmethods` here feel a bit odd. We should probably have a `PartialParameterExchanger` that inherit from `ParameterExchanger` and ` Generic[T]` similar to the `ParameterExhangerWithPacking`. Then we need not define the push/pull parameters here and just tack on the select_parameters requirement.

Reply:
Yes that's a good point. Does it also make sense then to not require this class to have `pack_parameters` or `unpack_parameters` methods? Otherwise it would be a bit strange to repeat those definitions here, although both of the partial exchangers we have need those methods...

Reply:
I think the way you implemented this looks good to me.